common/djangoapps/student/views.py

```diff
@@ -1804,6 +1805,30 @@ def create_account_with_params(request, params):
     return new_user
 
 
+def _enroll_user_in_pending_courses(student):
+    """
+    Enroll student in any pending courses he/she may have.
+    """
+    if not student:
+        log.error(u"There is no student")
```

Comment:
Yeah, you're right! `If not student:` I think there's no reason to do this check here, so I'll remove this condition.